### DISCUSSION TITLE
Questions about creating new theme

### DISCUSSION THREAD

**Discussion by DiegoSof2** [prasanthrangan/hyprdots#75] [General]
I am modifying a theme and I want it to be cohesive, just like the other themes in Hyprdots. I have read the theming.md file, and from what I understand, I need to create a theme for each listed application, including for GTK and QT. I have a few questions as I am a beginner in development.

Do I need to create themes for all versions of GTK (2, 3, 4)?
![image](https://github.com/prasanthrangan/hyprdots/assets/105827698/82ecf3b1-5d9a-4eb9-9742-8abe2b436ebe)
After completing the theme, should I post it here for approval and inclusion in the project? Or is the project concept already closed, and new themes are not accepted?

If new themes are not accepted, can I share my repository with my theme and the other hyprdots  themes?

I have an idea to change the Pokémon icons in Kitty to icons that match the theme I am creating. Is this possible?

what better place to report bugs?

**Comment by prasanthrangan** [comment]
> Do I need to create themes for all versions of GTK (2, 3, 4)?

Most current gtk apps use theme/colors set in `gtk-3.0` and `gtk4.0`. Its is very difficult to build these files from scratch so I would recommend to just copy any existing gtk theme (entire gtk folder) and replace the core color palette in `gtk-3.0` and `gtk4.0` file.

> After completing the theme, should I post it here for approval and inclusion in the project? Or is the project concept already closed, and new themes are not accepted?
> 
> If new themes are not accepted, can I share my repository with my theme and the other hyprdots themes?

If you create your theme, please feel free to share it by committing the files related to your theme, and I can approve/merge your theme to this repo.

> I have an idea to change the Pokémon icons in Kitty to icons that match the theme I am creating. Is this possible?

The pokemon sprites are launched in random from zshrc, so not sure if it can be matched with the theme.

> what better place to report bugs?

you could open an issue here detailing the bug.
